### PR TITLE
rhel: Be more explicit about system configuration

### PR DESCRIPTION
**Pull request by dm0-**

### CODE REVIEW

File: Documentation/install/rhel/installing-workers.md [hunk 1]
```diff
@@ -127,12 +127,14 @@ Note: These settings may not be all inclusive and will not represent relative no
 
 ### Set SELinux to Permissive mode
 
-It is required to run SELinux in Permissive mode. Running in Enforcing mode will block permissions for worker nodes.
+It is required to run SELinux in Permissive mode. Running in Enforcing mode will block permissions for worker nodes. The following command will switch to Permissive mode until reboot.
```
Comment (robszumski) [inline Documentation/install/rhel/installing-workers.md]: is there a good reason to recommend this at all? vs permanent solution?

Comment (dm0-) [reply Documentation/install/rhel/installing-workers.md]: If they just edit the configuration file, they would have to reboot before it works.

File: Documentation/install/rhel/installing-workers.md [hunk 2]
```diff
@@ -127,12 +127,14 @@ Note: These settings may not be all inclusive and will not represent relative no
...
 
 ```
-setenforce 0
+$ setenforce 0
 ```
 
+To boot into Permissive mode or with SELinux disabled completely, set `SELINUX=permissive` or `SELINUX=disabled` in `/etc/selinux/config`, respectively.
```
Comment (joshix) [inline Documentation/install/rhel/installing-workers.md]: s/, respectively//

Comment (dm0-) [reply Documentation/install/rhel/installing-workers.md]: I've dropped it, and I don't have permission to merge this.

**Pull request merged**